### PR TITLE
re-add per-consumer metrics

### PR DESCRIPTION
**Pull request by tangiel**

### CODE REVIEW

File: endpoints_management/control/report_request.py [hunk 1]
```diff
@@ -237,6 +240,7 @@ def __new__(cls,
                 auth_issuer=u'',
                 auth_audience=u'',
                 backend_time=None,
+                consumer_project_number=SIZE_NOT_SET,
```
Comment (inklesspen) [inline endpoints_management/control/report_request.py]: `SIZE_NOT_SET` doesn't make much sense as a "consumer project number". Can you perhaps rename the constant to `NOT_SET`?

File: endpoints_management/control/wsgi.py [hunk 2]
```diff
@@ -276,6 +276,7 @@ def __call__(self, environ, start_response):
         app_info.http_method = http_method
         app_info.url = parsed_uri
 
+        consumer_project_number = 0
```
Comment (inklesspen) [inline endpoints_management/control/wsgi.py]: Mark the meaning of this constant? (Via a comment, probably)

File: test/test_label_descriptor.py [hunk 3]
```diff
@@ -262,3 +263,8 @@ class SccServiceAgent(KnownLabelsBase, unittest2.TestCase):
 class SccUserAgent(KnownLabelsBase, unittest2.TestCase):
     SUBJECT = _KNOWN.SCC_USER_AGENT
     WANTED_LABEL_DICT = {SUBJECT.label_name: label_descriptor.USER_AGENT}
+
+
+class SccConsumerProject(KnownLabelsBase, unittest2.TestCase):
+    SUBJECT = _KNOWN.SCC_CONSUMER_PROJECT
+    WANTED_LABEL_DICT = {SUBJECT.label_name: "1234"}
```
Comment (inklesspen) [inline test/test_label_descriptor.py]: Please add trailing blank line.

Comment (tangiel) [reply test/test_label_descriptor.py]: Done.

File: endpoints_management/control/wsgi.py (hunk 2 above)
Comment (tangiel) [reply endpoints_management/control/wsgi.py]: Done.

File: endpoints_management/control/report_request.py (hunk 1 above)
Comment (tangiel) [reply endpoints_management/control/report_request.py]: Done.

**Pull request merged**